docs/advanced/payments/adyen.mdx

```diff
@@ -470,3 +516,17 @@ the Cart amount and the Store currency. Different contexts could display
 different payment methods in the checkout.
 
 :::
+
+# Payment methods support
+
+The following payment methods have been tested:
+
+- Credit card
+- Paypal
+- Klarna (Pay now, Pay over time, Pay later)
+- Google Pay
+- ApplePay
+- Sofort
+- Alipay: There is a known issue with Alipay when integrating Adyen through the
+  Magento 2 module
```

Comment:
is this comment about the adyen support request currently going on ?

Reply:
Yes. I wasn't planning on merging this until we have a proper answer (or the cycle ends).

Reply:
Confirmed it works after Adyen manual configuration. Updated the docs accordingly.